app/src/main/java/com/example/mymovieinfo/MainActivity.kt

```diff
@@ -2,11 +2,24 @@ package com.example.mymovieinfo
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+<<<<<<< Updated upstream
+=======
+import org.w3c.dom.Text
+import java.io.IOException
+import java.io.InputStream
+>>>>>>> Stashed changes
```

Comment:
Errrr... this is a merge conflict.  This shouldn't be pushed to GitHub, as it breaks the build.